documentation/omf-msg-sample.md

```diff
@@ -131,7 +136,8 @@ Create containers for the `dynamic` types.
 		"id": "Tank1Measurements",
 		"typeid": "TankMeasurement",
 		"indexes": ["Pressure"], 
-		"datasource":"Modbus"
+		"datasource":"Modbus",
```

Comment:
We might add a space between property and value here for consistency: "datasource": "Modbus",

Reply:
Done